tom/jenkins.py

```diff
@@ -78,29 +88,53 @@ def trigger(
             description = "{} @{} ({})".format(title, user, branches)
         else:
             description = "Unnamed build ({})".format(user)
-        if exotics:
-            description += " - WITH EXOTICS"
-        if no_tests:
-            params["NO_TESTS"] = True
-            description += " [NO TESTS]"
-        params["BUILD_DESC"] = description
-        if docs:
-            # building documentation
-            if params["BASE_BRANCH"] != "master":
-                params["BASE_BRANCH"] += ".x"
-            if "DOCS_REV" not in params:
-                params["DOCS_REV"] = str(branch)
-            if "DOCS_GEN_REV" not in params:
-                params["DOCS_GEN_REV"] = str(branch)
-            params["BUILD_DOCS"] = True
+        # both build-and-deploy-docs types (fast and regular) can use "pr" as the DOCS_BRANCH/BRANCH
+        # this translates to a docs URL of http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-pr-pipeline-52/output/_site/
+        # which includes after build-documentation-pr folder, a folder for the specific pipeline build
+        if "/build-and-deploy-docs" in path:
             params["DOCS_BRANCH"] = "pr"
-            params["NO_TESTS"] = True
-            params["NO_DEPLOYMENT_TESTS"] = True
-            params["NO_FR_TESTS"] = True
-            params["NO_STATIC_CHECKS"] = True
-            params[
-                "CONFIGURATIONS_FILTER"
-            ] = 'label == "PACKAGES_HUB_x86_64_linux_ubuntu_16"'
+        if "fast-build-and-deploy-docs" in path:
+            params["BRANCH"] = "pr"
+        repos_accepted = [
+            "core",
+            "enterprise",
+            "nova",
+            "masterfiles",
+            "documentation",
+            "documentation-generator",
+        ]
+        if "fast-build-and-deploy-docs" not in path:
+            repos_accepted = [
+                "libntech",
+                "core",
+                "enterprise",
+                "nova",
+                "masterfiles",
+                "buildscripts",
+                "mission-portal",
+                "ldap",
+                "mender-qa",
+                "documentation",
+                "documentation-generator",
+            ]
```

Comment:
It would be nice to do `repos_accepted.extend([...])` with the extra repos here. Just redefining the list makes the differences hard to see.

Reply:
Agreed. Will do.